README.rst

```diff
@@ -105,10 +105,9 @@ Manual Installation
         …
     ]
 
-   Please see notes regarding `Django CMS Requirements`_ and `Django Appdata`_
-   below, however.
+   Please see notes regarding `Django CMS Requirements`.
 
-3) Configure ``aldryn-boilerplates`` (https://pypi.python.org/pypi/aldryn-boilerplates/).
+3) Configure Aldryn Boilerplates (https://pypi.python.org/pypi/aldryn-boilerplates/).
 
    To use the old templates, set ``ALDRYN_BOILERPLATE_NAME='legacy'``.
    To use https://github.com/aldryn/aldryn-boilerplate-standard (recommended, will be renamed to
```

Comment:
yes it should be https://github.com/aldryn/aldryn-boilerplate-bootstrap3 instead of https://github.com/aldryn/aldryn-boilerplate-standard